recipe/meta.yaml

```diff
@@ -36,13 +38,25 @@ requirements:
 test:
   requires:
     - pip
+    # Running unit tests requires bottle, using flask instead via patch file.
+    - pytest >=5
+    - flask
+    - flaky
+    - pyflakes
+  source_files:
+    - pytest.ini
+    - README.rst
+    - setup.py
+    - tests
   commands:
     - pip check
+    # Skip tests for SSL backends we don't support.
+    - pytest -v tests -k "not (test_keyfunction or test_proxy_tlsauth or test_libcurl_ssl_gnutls or test_libcurl_ssl_nss or test_libcurl_ssl_openssl)"
```

Comment:
You could call `pytest -v -m standalone` (assuming standalone means not online, not ssh...)
https://github.com/pycurl/pycurl/blob/REL_7_45_3/pytest.ini#L9

Reply:
This appears to disable all but one test case.

Reply:
Hey, okay maybe not that then :)